Wangscape/CMakeLists.txt

```diff
@@ -33,3 +35,9 @@ if(SFML_FOUND)
     include_directories(${SFML_INCLUDE_DIR})
     target_link_libraries(Wangscape ${SFML_LIBRARIES})
 endif()
+
+find_package(Armadillo REQUIRED)
+if(SFML_FOUND)
```

Comment:
All caps according to https://github.com/Kitware/CMake/blob/master/Modules/FindArmadillo.cmake